File: Java/Car.java
class Car
{
    Integer id;
    String license;
    Account driver;
    Integer passenger;

    public Car(String license, Account driver)
    {
        this.license = license;
        this.driver = driver;        
    }
    {

    }

    void printDataCar()
    {        
        System.out.println("License: " + license.name);
        System.out.println("Driver: " + driver.name);
        System.out.println("Passenger: " + passenger);
    }
}

File: JavaScript/Car.js
class Car 
{
    constructor(license, driver) 
    {
        this.id;
        this.license = license;
        this.driver = driver;
        this.passenger;
    }
    printDataCar() 
    {
        console.log(`Driver: ${this.driver.name}, License: ${this.license}, Passenger: ${this.passenger}`);
    }
}